### PR TITLE
Add in debug discovery params

### PR DESCRIPTION
**Pull request by ckipp01**
As a follow-up to the discovery params added in here: https://github.com/scalameta/metals/pull/2532

### CODE REVIEW

**Review by tgodzik — APPROVED** [review]
LGTM! thanks!

File: src/interfaces/DebugDiscoveryParams.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export interface DebugDiscoveryParms {
```
Comment (tgodzik) [inline src/interfaces/DebugDiscoveryParams.ts:1]: Och no :( I never noticed the typo.  Well we can use that for 0.4.2 then :S

Comment (ckipp01) [reply src/interfaces/DebugDiscoveryParams.ts:1]: Whoops. We can also just fix and and do a quick release. This one is super quick, but if not no worries.

Comment (tgodzik) [reply src/interfaces/DebugDiscoveryParams.ts:1]: Haha, I think this is fine for now :P

**Pull request merged**